packages/callhierarchy/src/browser/callhierarchy-context.ts

```diff
@@ -0,0 +1,105 @@
+/********************************************************************************
+ * Copyright (C) 2018 TypeFox and others.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * This Source Code may also be made available under the following Secondary
+ * Licenses when the conditions for such availability set forth in the Eclipse
+ * Public License v. 2.0 are satisfied: GNU General Public License, version 2
+ * with the GNU Classpath Exception which is available at
+ * https://www.gnu.org/software/classpath/license.html.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
+ ********************************************************************************/
+
+import { ILanguageClient } from '@theia/languages/lib/browser';
+import {
+    ReferencesRequest, DocumentSymbolRequest, DefinitionRequest, TextDocumentPositionParams,
+    TextDocumentIdentifier, SymbolInformation, Location, Position, DocumentSymbol, ReferenceParams
+} from 'monaco-languageclient/lib/services';
+import * as utils from './utils';
+import { ILogger, Disposable } from '@theia/core';
+import { MonacoTextModelService } from '@theia/monaco/lib/browser/monaco-text-model-service';
+import URI from '@theia/core/lib/common/uri';
+
+export class CallHierarchyContext implements Disposable {
```

Comment:
I just extracted this code into a separate file because the tsserver does not like ;-) it's not reporting any issues.